src/include/hw.h

```diff
@@ -0,0 +1,15 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+#pragma once
+
+/*
+ * -- __wti_hw_proc_freq_hz
+ *      Return the number of nanoseconds per cpu "tick". If the value is unknown
+ *      the function returns 0.
+ */
+uint64_t __wti_hw_proc_freq_hz(void);
```

Comment:
We already have a src/include/hardware.h, having hw.h as well is confusing.  But see my other comment, we may not need this declaration anyway.

Reply:
I wanted to separate this out for the purposes of discussion given I assumed it would have to be changed (weak symbol discussion). It's probably better moved into the os porting layer as windows provide a system call for querying such information on x86.